AcademyApp/Info.plist

```diff
@@ -2,6 +2,14 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Maison_Neue_Medium.otf</string>
```

Comment:
Are all imported fonts needed? I found that just 3 of them are used

Reply:
Removed imported fonts which are not used.